test/functional/test-performance.js

```diff
@@ -14,7 +14,11 @@
  * limitations under the License.
  */
 
-import {installPerformanceService} from '../../src/service/performance-impl';
+import * as sinon from 'sinon';
```

Comment:
When I merged master, the import was removed but there's still a reference to it on line 449:
https://github.com/kmh287/amphtml/blob/4853b2ce52c52b4f628c800b14da338bd486b7a6/test/functional/test-performance.js#L449